python/python/tests/utils/utils.py

```diff
@@ -359,3 +361,40 @@ def check_function_stats_response(
         b"LUA": {b"libraries_count": lib_count, b"functions_count": function_count}
     }
     assert expected == response.get(b"engines")
+
+
+def ft_search_deep_compare_result(
```

Comment:
I would keep this in the same file as the test (test_ft_search) since it looks very closely related to the ft.search test, and won't be very useful for any other tests.

Reply:
TBH, all this code could be included in the test.  I'm not sure what the value is anybody to having a separate function.

Reply:
Updated.